README.md

```diff
@@ -101,6 +101,15 @@ $ kubectl get no -w
 **Note:-** You should be seeing nodes joining the cluster within less than minutes.
 
 ---
+#### EKS cluster upgrade using new asg file in terraform
+Create a new eks-worker-node-v1.tf file with different name and below changes you have to do for EKS cluster upgrade.
+* Change the userdata name to new version(eks-worker-node-upgrade-v2.tf) and should not conflict with old one.
+* Change the Launch configuration and autoscalling group name to new version and should not conflict with old one.
+* Change the ami to which your going upgrade EKS version provided by AWS -- ##eks-worker-ami -- change to new version  
+* And in the new worker node file(eks-worker-node-upgrade-v2.tf), we have updated how to use taint for dedicated node.
```

Comment:
Change this like to:

In the new worker node file (`eks-worker-node-upgrade-v2.tf`), we have updated how to use taint for a dedicated node.